packages/piro/src/Piro_TempusSolver_Def.hpp

```diff
@@ -167,24 +167,10 @@ void Piro::TempusSolver<Scalar>::initialize(
     const std::string stepperType = stepperPL->get<std::string>("Stepper Type", "Backward Euler");
     //*out_ << "Stepper Type = " << stepperType << "\n";
 
-    //
```

Comment:
The rm-ed lines where for stuff _never_ used (directly or indirectly). Probably a leftover from earlier dev cycles?